libs/qrencode/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2012 OpenWrt.org
+# Copyright (C) 2006-2017 OpenWrt.org
```

Comment:
no. You're not openwrt.org and can't claim copyrigth on it's behalf, and the general form of it (adding a change 5-6 years after the fact) would be 2006-2012,2017 anyway.